gatsby-node.js

```diff
@@ -11,6 +11,7 @@ const fs = require('fs');
 const template = require('lodash/template');
 const wrap = require('lodash/wrap');
 const { availableLocales, defaultLocale } = require('./intl/config');
+const ServiceWorkerWebpackPlugin = require('serviceworker-webpack-plugin')
```

Comment:
There's a few linting errors, do you have your editor setup correctly?

Reply:
Yeah! I am using [Standardjs linting](https://standardjs.com/), which is used by all the IPFS community projects. I will change to your linting style for now.

The guidelines are [here](https://github.com/ipfs/community/blob/master/js-code-guidelines.md#linting--code-style). I think this project will need to follow them as well.

Reply:
I see. Please change it for now and we will transform all the code to be compatible with standardjs later on. https://github.com/ipfs/js.ipfs.io/issues/5